src/emu/console/src/thread.cpp

```diff
@@ -45,11 +45,41 @@ void set_mouse_down(void *userdata, const int button, const bool op) {
     emu->mouse_down[button] = op;
 }
 
+static eka2l1::drivers::input_event make_mouse_event_driver(float x, float y, int button, int action) {
+    eka2l1::drivers::input_event evt = *new eka2l1::drivers::input_event();
```

Comment:
Don't do this. 

```suggestion
    eka2l1::drivers::input_event evt;
```

That leaks memory